Packs/SophosCentral/ReleaseNotes/1_3_2.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### Sophos Central
+- Added new argument to sophos-central-endpoint-list to query endpoints by IP hostname
+- Added new argument to sophos-central-endpoint-list to check hostname for full match instead of partial (default API behavior)
```

Comment:
```suggestion
- Added new argument to ***sophos-central-endpoint-list*** to check hostname for full match instead of partial (default API behavior)
```